### ISSUE TITLE
Run YOLO detection models within the survey start and survey end

### ISSUE THREAD

**Issue by victor-wildlife** [ocean-data-factory-sweden/kso#341]
At the moment tut#9 runs all the way from the beginning of the video to the end. We need to implement a way to check the sampling_start and sampling_end points of the movies and run the detection for the frames between those two points

**Comment by zitaker** [comment]
I'm working on it

**Comment by jannesgg** [comment]
@victor-wildlife I don't think it is currently practical to run detection only over sections of video. Instead, we can run the detection over the whole video (which means we don't need to cut it up or create any additional files), and then filter the output instead, e.g. the csv of detections we get out has the frame_no as a column, and we just filter by min and max? 

Does that seem workable?

**Comment by victor-wildlife** [comment]
yeah, that approach will fix the issue, could you implement it?
